test/parallel/test-preload.js

```diff
@@ -8,9 +8,9 @@ var nodeBinary = process.argv[0];
 
 var preloadOption = function(preloads) {
   var option = '';
-  preloads.forEach(function(preload, index) {
-    // TODO: randomly pick -r or --require
-    option += '-r ' + preload + ' ';
+  preloads.forEach(function(preload) {
+    var optionString = ['-r', '--require'][Math.floor(Math.random()*2)];
```

Comment:
@brendanashworth The edge case I was thinking of is `node -r module1 --require module2`. Thinking about it now, that can probably just be dropped. We don't test combinations of other command line flags--they just get tested individually.

So maybe just go with a simplified version of @dcousens's suggestion and wrap it in `[`-r', '--require'].forEach()`?